Model/Payment.swift

```diff
@@ -0,0 +1,111 @@
+// This file is part of Kiwix for iOS & macOS.
+//
+// Kiwix is free software; you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 3 of the License, or
+// any later version.
+//
+// Kiwix is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Kiwix; If not, see https://www.gnu.org/licenses/.
+
+import Foundation
+import PassKit
+import SwiftUI
+import Combine
+
+struct Payment {
+
+    let completeSubject = PassthroughSubject<Bool, Never>()
+
+    static let merchantId = "merchant.org.kiwix.apple"
+    static let paymentSubscriptionManagingURL = "https://www.kiwix.org"
+    static let supportedNetworks: [PKPaymentNetwork] = [
```

Comment:
Where is this from? Is this from Stripe documentation? The fact that we are using a third party (Stripe) makes me think we shouldn't need those things

Reply:
Stripe is not yet added, this form comes from Apple's PassKit framework.